vspace/src/main/java/edu/asu/diging/vspace/web/staff/SpaceController.java

```diff
@@ -29,14 +35,23 @@
 
     @RequestMapping("/staff/space/{id}")
     public String showSpace(@PathVariable String id, Model model) {
+    	
+    	
         ISpace space = spaceManager.getFullyLoadedSpace(id);
+        model.addAttribute("isSpaceLinkPresent", spaceManager.isLinkedFromSpace(id));
```

Comment:
`isLinkedFromSpace` returns a list, right? so the attribute should have an appropriate name. right now it sounds like it contains a boolean.